web_src/less/_base.less

```diff
@@ -393,6 +394,11 @@ a.commit-statuses-trigger {
   background: var(--color-grey);
 }
 
+.ui.active.label {
+  background: var(--color-label-active);
```

Comment:
Should we set the border color as well as it was done previously (even though it was set to the same color)?

Reply:
IIRC, these do not have a border.

Reply:
https://github.com/go-gitea/gitea/blob/3d92ad8eb9cc255f8153d40e5cbc6a0e9f8fc9eb/web_src/less/themes/theme-arc-green.less#L277

Reply:
Ah I missed that rule, need to remove it.

Reply:
You removed it already.

Reply:
Border added. I think those rules are most likely pointless as `.ui.label` has a border value of `0 solid transparent`, but I guess to be safe, we override those rules as well.